packages/client/hmi-client/src/types/Types.ts

```diff
@@ -686,7 +687,8 @@ export interface EnsembleSimulationCiemssRequest {
 export interface OptimizeRequestCiemss {
     modelConfigId: string;
     timespan: TimeSpan;
-    interventions?: OptimizedIntervention;
+    policyInterventions?: OptimizedInterventions;
+    fixedStaticParameterInterventions?: Intervention[];
```

Comment:
Food for thought for future improvement, It seems like we should consider consolidate the two intervention objects to make things a bit clearer and easy to work with? 

An interventions array []:
```
export interface Intervention {
    name: string;
    timestep: number;
    value: number;
}
```

And an intervention object with parallel arrays:
```
export interface OptimizedInterventions {
    selection: string;
    paramNames: string[];
    paramValues?: number[];
    startTime?: number[];
}
```

Reply:
totally see your point. I will need to look into what changes will need to be made in the pyciemss-service for this.
This should probably be a future improvement as you suggest as it may clutter these PRs
Ive made the following to not forget:
https://github.com/DARPA-ASKEM/terarium/issues/3711